File: 4st/Eunseong/날짜 비교하기.java
class Solution {
    public int solution(int[] date1, int[] date2) {
        for (int i = 0; i < date1.length; i++) {
            if (date1[i] < date2[i]) {
                return 1;
            } else if (date1[i] > date2[i]) {
                break;
            }
        }
        return 0;
    }
}

File: 4st/Hani/programmer/날짜 비교하기.py
def solution(date1, date2):
    if date1[0] < date2[0] :
        return 1
    elif date1[0] == date2[0] and date1[1] < date2[1] :
        return 1
    elif date1[0] == date2[0] and date1[1] == date2[1] and date1[2] < date2[2] :
        return 1
    else :
        return 0